### PR TITLE
autogenerate invoice number & bugfixes

### PR DESCRIPTION
**Pull request by apoorv1316**
What
- Implemented autogenerate invoice number for mobile
- Bugfixes in clientSelction on both web & mobile
- Refactored the client_list partial

### CODE REVIEW

File: app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx [hunk 1]
```diff
@@ -49,6 +52,61 @@ const InvoiceDetails = ({
     setDueDate(newDueDate);
   };
 
+  const list = companyClientList || clientList;
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx:55]: can you renames list to something with client

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]
Please delete `companies/_client_list.json.jbuilder`

File: app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx (hunk 1 above)
Comment (apoorv1316) [reply app/javascript/src/components/Invoices/Generate/MobileView/Container/MenuContainer/InvoiceDetails.tsx]: done.

**Comment by apoorv1316** [comment]
@prasanthchaduvula I've fixed it. Please check.

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**